compiler/rustc_errors/src/lib.rs

```diff
@@ -16,12 +16,17 @@
 #![feature(error_reporter)]
 #![allow(incomplete_features)]
 
+//#![deny(rustc::untranslatable_diagnostic)]
+//#![deny(rustc::diagnostic_outside_of_impl)]
```

Comment:
Because of some crates internally uses non-translated diagnostic API like span_labels, it's not able to deny those.